src/main/java/microsoft/exchange/webservices/data/AttachmentCollection.java

```diff
@@ -379,7 +377,9 @@ public void validate() throws Exception {
             contactPhotoFound = true;
           }
         }
-        attachment.validate(attachmentIndex);
+        if (null != attachment) {
```

Comment:
this check is already mentioned in [L357](https://github.com/ahaarrestad/ews-java-api/blob/FirstFindbugsCleanup/src/main/java/microsoft/exchange/webservices/data/AttachmentCollection.java#L357)